checkers/DBChecker.go

```diff
@@ -0,0 +1,47 @@
+package checkers
+
+import (
+	"database/sql"
+	"errors"
+)
+
+// DBChecker sturct provide health check for SQL DB by `sql.DB`
+type DBChecker struct {
+	HealthChecker HealthChecker
+	QuerySQL      string
+	DB            *sql.DB
+}
+
+// NewDBChecker return new instance DBChecker with name and DB resource args
+func NewDBChecker(name string, DB *sql.DB) *DBChecker {
+	return &DBChecker{
+		HealthChecker: NewHealthChecker(name),
+		QuerySQL:      "SELECT 1",
+		DB:            DB,
+	}
+}
+
+// Check health DB service
+func (dbc *DBChecker) Check() (Health, error) {
```

Comment:
Can return an error, not bool? so we will find out why the service is gone.

Reply:
Yes, that right.
But I return an error, or I not right understand you.